.github/workflows/test.yml

```diff
@@ -0,0 +1,72 @@
+name: Unit Test
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'v4'
+      - 'feature**'
+  delete:
+    branches:
+      - 'feature**'
+
+
+
+env:
+  SAM_TEMPLATE: template.yaml
+  TESTING_STACK_NAME: pdf-encrypt-dev
+  TESTING_REGION: ap-northeast-1
+  PROD_STACK_NAME: pdf-encrypt-prod
+  PROD_REGION: ap-northeast-1
+
+permissions:
+  id-token: write
+  contents: read
+jobs:
+
+  test:
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v3
+      - name: bash test
+        shell: bash
+        run: |
+            # trigger the tests here
+
+      - name: Test encrypt-pdf function locally
+        run: |
+
+          cd unit_test
+          pip install pytest
+          pip install -r requirements.txt
+          pytest -s -v
+          cd ..
+
+  delete-feature:
```

Comment:
@javanlth this pipeline is supposed to trigger your tests like pytest - not sure why the delete-feature is here?

Reply:
OK will fix it. Was default from AWS SAM's constructor.